api-admin/src/main/java/bio/terra/pearl/api/admin/controller/enrollee/EnrolleeController.java

```diff
@@ -45,6 +50,21 @@ public ResponseEntity<Object> listChangeRecords(
     return ResponseEntity.ok(records);
   }
 
+  @Override
+  public ResponseEntity<Object> updateProfile(
```

Comment:
I would put this in a separate ProfileController -- Profiles aren't 'owned' by the Enrollee, they're owned by the PortalParticipantUser.  And once we have proxies we'll need to edit Profiles that aren't attached to enrollees